src/main/kotlin/participant/Dealer.kt

```diff
@@ -0,0 +1,10 @@
+package participant
+
+import card.PlayingCard
+import state.State
+
+class Dealer(name: String = "Dealer", override var state: State) : Participant(name) {
+    override fun showCardFirst(): List<PlayingCard> {
+        return listOf(state.hand.cards.first())
```

Comment:
This line reaches through multiple objects, which slightly violates the _Law of Demeter_.
You might consider exposing a function to encapsulate this access and make the intention clearer!

This is similar to what I mentioned in a previous comment: https://github.com/next-step/kotlin-blackjack/pull/833#discussion_r2070934966